src/components/Home/Ritika/Week3/utils/constants.json

```diff
@@ -3,4 +3,5 @@
     "autoFocusInputWarning":"Please provide both name and phone number.",
     "alreadyExistUser":"Employee with this name already exists!",
     "successMessage":"Employee added successfully"
+    
```

Comment:
Why this extra line here?

Reply:
this is the eol, an eslint feature

Reply:
eol should be lastline (ie after line 8) not before here

Reply:
Oh okay , my mistake shree, may be I did the changes with an extra line and pushed the code. But I had resolved this comment as well